build-image/first-boot.bash

```diff
@@ -18,6 +19,30 @@ fail_inprogress() {
   exit 1
 }
 
+## Attempt to update apt package lists 10 times
+## unless 'apt-get update' evaulates to 0.
+## Sleeps for 1 second between each attempt.
+##
+##    wait_for_apt_to_be_ready()
+##
+wait_for_apt_to_be_ready() {
```

Comment:
I'd move this function and the other apt_ related ones to helpers.bash
That is inserted into first-boot.bash on image creation so we don't need to duplicate code

Reply:
Done.